pandas/core/frame.py

```diff
@@ -5480,7 +5528,7 @@ def sort_values(  # type: ignore[override]
         )
 
         if ignore_index:
-            new_data.axes[1] = ibase.default_index(len(indexer))
+            new_data.set_axis(1, ibase.default_index(len(indexer)))
```

Comment:
i think there's a set_axis_without_check?

Reply:
I don't find a method called like that

Reply:
NDFrame._set_axis_nocheck

Reply:
> NDFrame._set_axis_nocheck

But that's a DataFrame method, the line you are commenting on is manager?

Reply:
makes sense, thanks